src/main/plugins/org.dita.eclipsehelp/build_dita2eclipsehelp_template.xml

```diff
@@ -26,14 +29,21 @@ See the accompanying LICENSE file for applicable license.
 	
     <target name="dita2eclipsehelp"
             unless="noMap"
-        depends="build-init, dita.eclipsehelp.init, preprocess, 
+        depends="dita.eclipsehelp.init, build-init, preprocess, 
                  xhtml.topics,
                  copy-css">
       <antcall target="dita.map.eclipse"></antcall>
+      <jar destfile="${output.dir}/${args.eclipsehelp.jar.name}" basedir="${dita.output.dir}" if:set="args.eclipsehelp.jar.name">
```

Comment:
Agreed, just didn't want to touch stuff that has been there for years.

If we remove that `<antcall>` and just move `dita.map.eclipse` into the `dita2eclipsehelp` dependency list, is there any chance of side effects? I wouldn't think so but I don't know why it was chosen in the first place.